CHANGELOG.md

```diff
@@ -3,6 +3,74 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.12.1
+
+_Aug 8, 2024_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎨 Charts get a new component to display color mapping in the legend.
+- 🚀 The `@mui/x-charts-pro` is released in alpha version 🧪. This new package introduces two main features:
+  - The Heatmap component
+  - The zoom interaction on the bar, line, and scatter charts
```

Comment:
```suggestion
  - The [Heatmap component](https://mui.com/x/react-charts/heatmap/)
  - The [zoom interaction](https://mui.com/x/react-charts/zoom-and-pan/) on the bar, line, and scatter charts
```

Reply:
Also, should it be a minor version bump since we release new features for Charts?
cc @alexfauquette

Reply:
Those features are available only on the `-pro` package, which is being released only now and as `alpha.0`.
Not sure if that warrants a minor bump. 🤔

Reply:
Afaict pre-releases fall under the same category as `0.x` releases, aka "do whatever you want", in other words they fall outside of semver guarantees. That's how I understand the spec: https://semver.org/#spec-item-9